components/Repositories/RossChart.tsx

```diff
@@ -45,7 +45,11 @@ export default function RossChart({ stats, isLoading, error, range, onFilterClic
   }, [stats]);
 
   const weeklyData = useMemo(() => {
-    return stats?.contributors.reverse().map((week) => {
+    // @ts-ignore
```

Comment:
`Array.toReversed()` has been supported by all major browsers since last year; I don't know why Typescript/tsserver do not recognize it. Adding `@ts-ignore` to remove build errors

Reply:
> I don't know why Typescript/tsserver do not recognize it

I wonder if we have an older version of those types / tools that don't yet have full GA support for it yet. Regardless, this is fine 👍🏼 I'd maybe add a small comment for future us to remove that `ts-ignore` when we consume a version of the tooling that supports this

Reply:
`ts-ignore` may also just be taking the conservative approach: while it's been supported for well over a year, not everyone upgrades their browsers that often. Still, I'd say this is fine!

Reply:
Would using `slice` before doing the `reverse` help?  As `reverse` mutates the array in place, where `slice` would create a new copy first.

Reply:
Yeah if we want to remove the `@ts-ignore`, `slice` then reverse would do it. I'll get a quick commit in for it